pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v3/Transactions.java

```diff
@@ -55,6 +55,55 @@ public void getCoordinatorStatus(@Suspended final AsyncResponse asyncResponse,
         internalGetCoordinatorStatus(asyncResponse, authoritative, coordinatorId);
     }
 
+    @GET
+    @Path("/transactionInBufferStats")
+    @ApiOperation(value = "Get transaction state in transaction buffer.")
+    @ApiResponses(value = {@ApiResponse(code = 403, message = "Don't have admin permission"),
+            @ApiResponse(code = 404, message = "Tenant or cluster or namespace or topic doesn't exist"),
+            @ApiResponse(code = 503, message = "This Broker is not configured "
+                    + "with transactionCoordinatorEnabled=true."),
+            @ApiResponse(code = 307, message = "Topic don't owner by this broker!"),
+            @ApiResponse(code = 501, message = "Topic is not a persistent topic!"),
```

Comment:
"Topic is not a persistent topic"

501 is kind of "internal server error", probably we should return something like "Not allowed" or "Bad request"